marlowe/specification/ReadMe.md

```diff
@@ -0,0 +1,346 @@
+# On-Chain Transaction Specification for Marlowe
+
+
+## Scope
+
+This document defines the specification for Marlowe semantics's interface with the Cardano blockchain. Marlowe utilizes three Plutus scripts: (i) a monetary policy for the roles currency used in some Marlowe contracts; (ii) the Marlowe application validator script that enforces Marlowe semantics; and (iii) the Marlowe payout-validator script that allows the holder of a role token to withdraw funds paid by the Marlowe application.
+
```

Comment:
Would be useful to have an overview of what the document contains. E.g. a list of sections.

Reply:
Thanks, I'll expand this and add better links to the Isabelle specification.

Reply:
fixed in 8be04323e4ec24c4e923ed214094e85c916ab8e0